src/DefaultBuilder/src/BootstrapHostBuilder.cs

```diff
@@ -1,9 +1,11 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+using System.Reflection;
```

Comment:
Is this used anywhere?

Reply:
Yep https://github.com/dotnet/aspnetcore/pull/34829/files#diff-f0e6f715f99a336de90bdbeef327cb71b7ee811d00d7c1722049557adaa009c5R44